### ISSUE TITLE
Unable to delete file appengine-api-1.0-sdk-1.9.28.jar

### ISSUE THREAD

**Issue by gfcut** [GoogleCloudPlatform/gradle-appengine-templates#61]
Hi,
Whenever I try to run my app, after building the backend module, I get this error.
Is there a workaround for this?

Thanks

Error:Execution failed for task ':backend:clean'.
> Unable to delete file: C:\Users\gratian.cutulescu\git\Investigations\backend\build\exploded-app\WEB-INF\lib\appengine-api-1.0-sdk-1.9.28.jar

**Comment by rahulfhp** [comment]
Is your backend App Engine server in start mode when this error occur ?

Regards
Rahul Purswani
Founder & Developer - My Android App - CricFriends
<https://play.google.com/store/apps/details?id=com.mindefy.cricfrnds.app&hl=en>

**Comment by gfcut** [comment]
Yes. Is it OK to perform a backend:clean task while the backend is running?

**Comment by rahulfhp** [comment]
For this reason only it gives this error, as it keeps a lock on it .

Ideally you should stop the backend and then perform clean operation .

Regards
Rahul Purswani
Founder & Developer - My Android App - CricFriends
<https://play.google.com/store/apps/details?id=com.mindefy.cricfrnds.app&hl=en>

**Comment by gfcut** [comment]
I had to configure my run task for the app. Now it performs only: assembleDebug and installDebug. This means that there won't be a backend:clean task performed. If this is the solution, this should be documented.

**Comment by 8Ozymandias** [comment]
@gfcut  I am experiencing the same thing. The only difference with me is that I am dealing with:
 
appengine-api-1.0-sdk-1.9.42.jar

**Comment by 8Ozymandias** [comment]
@rahulfhp I tried what you suggested and it worked. I stopped the server, went to build and clicked clean project and there are now 0 errors.

**Comment by 8Ozymandias** [comment]
@gfcut  I don't understand what you're saying here:

"I had to configure my run task for the app. Now it performs only: assembleDebug and installDebug. This means that there won't be a backend:clean task performed. If this is the solution, this should be documented."
